.gitattributes

```diff
@@ -1,17 +1,18 @@
+data/pancan_scaled_zeroone_rnaseq.tsv.gz filter=lfs diff=lfs merge=lfs -text
```

Comment:
Rather than specifying ever single file to track with LFS, I usually just specify the type of file. Usually this will be `*.gz` or some other compression extension. Therefore by compressing a file, I am also signalling for it to use LFS.

Reply:
I will keep this in mind as I move forward. Currently, there are data in the repository that I don't want automatically tracked.

Reply:
> Currently, there are data in the repository that I don't want automatically tracked.

This file has NOTHING TO DO WITH WHETHER A FILE GETS TRACKED!!!!! `.gitignore` controls that!!!!!

Reply:
lol yeah. I guess I'm saying that its my preference to add these files to git LFS like this for now. I use the strategy you suggested in a [different private data repository](https://github.com/greenelab/aaces_ovarian_data/blob/master/.gitattributes).